config.go

```diff
@@ -5,23 +5,14 @@ package main
 
 import (
 	"bytes"
-	"errors"
 	"fmt"
-	"io"
-	"io/fs"
-	"strings"
 
 	"github.com/tidwall/gjson"
 )
 
-type rule struct {
-	inline  string
-	include string
-}
-
 // pluginConfiguration is a type to represent an example configuration for this wasm plugin.
 type pluginConfiguration struct {
-	rules []rule
+	rules string
```

Comment:
From @anuraaga in [this comment](https://github.com/anuraaga/coraza-wasm-filter/pull/31#discussion_r974798351):

Looks like it would be good to support `[]string`, either multiple strings or multiline string would work, former looks better in JSON maybe.